generators/package/index.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+var generators = require('yeoman-generator');
+var npmName = require('npm-name');
+var path = require('path');
+var slugify = require("underscore.string/slugify");
+
+module.exports = generators.Base.extend({
+	initializing: function () {
+		this.fs.copy(
```

Comment:
Also, to process a template we need to use `this.fs.copyTpl()` and pass manually the template context we'll be using (as the third parameter).